packages/react-core/src/components/ContextSelector/ContextSelector.tsx

```diff
@@ -102,7 +102,7 @@ export class ContextSelector extends React.Component<ContextSelectorProps, { oui
   popperRef: React.RefObject<HTMLDivElement> = React.createRef();
 
   onEnterPressed = (event: any) => {
-    if (event.charCode === KEY_CODES.ENTER) {
+    if (event.key === KeyTypes.Enter) {
```

Comment:
oh - maybe this is the constant I was imagining. Could we uses these/expand these everywhere you have hardcoded KeyTypes?

Reply:
Yeah I really like that idea!

Since RTL requires those `{}` around the actual key value though I would either need to either place the KeyTypes value in a template string with them or create a new constant for RTL keys, which approach do you think would be better?

Reply:
if the use of the template string means that we'd have one 'gold source' for key values, then I think it'd opt for that.